0085-Hex-Coverage-Limit.md

```diff
@@ -0,0 +1,89 @@
+ 
+## Detailed Explanation:
+Currently, any redundant and overlapping network coverage is still rewarded the same as non-overlapping coverage. This discourages the buildout of coverage to new areas. To prevent overcrowding and overlapping of coverage in hexes, this HIP proposes to limit the amount of modeled coverage points radios receive for redundant coverage in res12 hexes. 
+
+To ensure that only the best setups are rewarded, only the top two (2) ranked radio signals in each res12 hex will be awarded modeled coverage points, with a decaying multiplier based on the radio rank noted below. Any radios not ranked within the top two (2) will be ranked as “Fail”, and receive no modeled coverage points for that res12 hex. 
+
+| Rank         |Multiplier|  
+|--------------|----------|
+|      1       |   1X     |
+|      2       |  .75X    |
+|    Fail      |   0X     |
+
+Please note, that the multiplier table above only affects the modeled coverage points that are given to each outdoor radio, and does not affect rewards distributed for the transfer of data. 
+
+### Radio Ranking & Criteria
+
+All outdoor radios that provide coverage to any res12 hex will be given a rank for each res12 hex they provide coverage in based on the following potential attributes (note, this rank is only for a single res12 hex and not the entire radio):
+
+- Modeled Signal Strength 
+
+- Heartbeat Steak - The Heartbeat Streak is the length of time that has elapsed since the first heartbeat of that radio at its current location, that has had at least 1 heartbeat every 72 hours. If a radio does not produce at least one (1) heartbeat over 72 consecutive hours, the streak is reset. This attribute is only used as a tiebreaker in instances where two or more radios tie for Modeled Signal Strength. If somehow, two or more radios tie both Modeled Signal Strength and Heartbeat Streak, the radio to be ranked will be randomly selected every epoch.
+
+### Modeling Radio Ranking
+
+See the example below of how ranking based on a hex multiplier would affect a deployment of five (5) outdoor radios if they are providing modeled coverage to the same res12 hex:
+
+| Radio |Signal Strength| Heartbeat Streak Start Date | Rank  | Coverage Points Per HIP 74| New Coverage Points|  
+|-------|---------------|-----------------------------|-------|---------------------------|--------------------|
+|   A   |   -77.33 dBm  |05/01/2023 23:24:25          | 1     | 16 (16 * 1)               | 16  (16 * 1)       |
+|   B   |   -88.75 dBm  |12/01/2022 01:01:01          | 2     | 16 (16 * 1)               | 12  (16 * .75)     |
+|   C   |   -88.75 dBm  |12/02/2022 12:11:01          | Fail  | 16 (16 * 1)               | 0   (16 * .5)      |
+|   D   |   -93.60 dBm  |12/05/2022 11:51:01          | Fail  | 16 (16 * 1)               | 0   (16 * 0)       |
+|   E   |   -94.69 dBm  |08/01/2022 05:01:59          | Fail  | 16 (16 * 1)               | 0   (16 * 0)       |
+
+Table Key:
```

Comment:
Added this section. It should help clarify what this table is giving an example of and what you're trying to get across.

Reply:
@hiptron I have the table example/description right below ### Modeling Radio Ranking header. Did you want me to move it below the Table Key?

Also, all other comments should be addressed.

Reply:
I'll add some bullet points to fix it.